### PR TITLE
feat: add Moonbeam Testnet chain option

### LINKED ISSUE #45: Add Moonbeam Testnet bridge option
# Description

Add moonbeam option in Testnet

### PR DESCRIPTION
**Pull request by raop155**
# Description

This PR add `Moonbeam` Alpha Testnet chain option
Fix #45 

- Add support for Wallet Connect

---

Need to be test:

- Metamask
- Wallet Connect

### CODE REVIEW

**Review by evan-gray — CHANGES_REQUESTED** [review]
The changes here seem to be correct, but I think it's missing the ShowTx, SmartAddress, and TransactionProgress changes like https://github.com/wormhole-foundation/example-token-bridge-ui/pull/23/files. Could you please add those too?

File: src/utils/consts.ts [hunk 1]
```diff
@@ -479,6 +496,22 @@ export const NEAR_CORE_BRIDGE_ACCOUNT =
 export const NEAR_TOKEN_BRIDGE_ACCOUNT =
   CLUSTER === "testnet" ? "token.wormhole.testnet" : "token.test.near";
 
+export const MOONBEAM_BRIDGE_ADDRESS = getAddress(
```
Comment (kev1n-peters) [inline src/utils/consts.ts]: Can we remove these? They appear to be unused

File: src/utils/metaMaskChainParameters.ts [hunk 2]
```diff
@@ -118,6 +118,13 @@ export const METAMASK_CHAIN_PARAMETERS: {
     rpcUrls: ["https://testnet.aurora.dev"],
     blockExplorerUrls: ["https://testnet.aurorascan.dev"],
   },
+  1287: {
+    chainId: "0x507",
+    chainName: "Moonbase Alpha",
+    nativeCurrency: { name: "Dev", symbol: "DEV", decimals: 18 },
```
Comment (kev1n-peters) [inline src/utils/metaMaskChainParameters.ts:124]: I think the native currency should be GLMR

Comment (raop155) [reply src/utils/metaMaskChainParameters.ts:124]: @kev1n-peters
I can do that, that was my first choice but when I searched for the explorer:
https://moonbase.moonscan.io/

They use `DEV` as their main currency.

But I'll do the change.

Comment (kev1n-peters) [reply src/utils/metaMaskChainParameters.ts:124]: Actually I see it as DEV in the explorer so 🤷‍♂️

File: src/utils/consts.ts (hunk 1 above)
Comment (raop155) [reply src/utils/consts.ts]: Done

**Review by kev1n-peters — APPROVED** [review]

**Review by evan-gray — APPROVED** [review]

**Pull request merged**